docs/guide.rst

```diff
@@ -609,8 +609,7 @@ agent(s)?
 
 **treatment_anticoagulation**: Bool. Treated with anticoagulants during
 the visit? (Note - this is often combines a general anticoagulation
-field with those specific to Heparin. NOTE TO DEVS: merge (if not
-already) with treatment_indication_anticoagulation)
+field with those specific to Heparin.
```

Comment:
Did we merge the two attributes "treatment_indication_anticoagulation" and "treatment_anticoagulation"? or are they still separate?

Reply:
treatment_indication_anticoagulation is unused in previous parsers at this point, so it's been removed as they mean the same thing.